templates/sfn/ingest_sfn_definition.json.tpl

```diff
@@ -553,7 +539,7 @@
               "Entries": [
                 {
                   "Detail": {
-                    "slackMessage.$": "$.reason"
+                    "slackMessage.$": ":alert-noflash-slow: Reconcile failed for asset $.assetId"
```

Comment:
1. Maybe "Reconciliation" rather than "Reconcile"?
2. Is it possible/Should we add the bucket/Object path to this message so that the output could be found easier?

Reply:
1. Ok
2. It's a bit fiddly. The path in S3 of the results is `reconcilerOutput/<map-run-uuid>` but it's not trivial to get that ID, you have to set it yourself and then pass it through with the standard input which seems like overkill. I'll put a line in the slack message to say "See the state output for the result key." and we'll see how we get on with that.